docroot/sites/admissions.uiowa.edu/modules/admissions_core/admissions_core.module

```diff
@@ -106,7 +107,21 @@ function admissions_core_node_presave(Node $node) {
  * Implements hook_preprocess_HOOK().
  */
 function admissions_core_preprocess_layout(&$variables) {
+  /** @var \Drupal\Core\Layout\LayoutDefinition $layout */
+  $layout = $variables['layout'];
 
+  // Process the header layout.
+  if ($layout->id() === 'layout_header') {
+    $node = \Drupal::routeMatch()->getParameter('node');
+    $node = (isset($node) ? $node : \Drupal::routeMatch()->getParameter('node_preview'));
+    // Limit to just Student profiles.
+    if ($node instanceof NodeInterface && $node->getType() === 'student_profile') {
+      if ($node->hasField('field_image') && !$node->get('field_image')->isEmpty()) {
+        // Set the featured image display.
+        $variables['featured_image_display'] = 'medium';
```

Comment:
@pyrello I think we want this set to `large` instead of `medium`.

Reply:
Okay. I was doing this from memory, so I didn't recall that this was using the 'large' version. I'm in another PR at the moment, so feel free to change this prior to merging if you are able to.